CHANGELOG/CHANGELOG-4.0.md

```diff
@@ -3,7 +3,7 @@
 Previous change logs can be found at [CHANGELOG-3.x](https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.x.md).
 
 
-The minimum recommended etcd versions to run in **production** are 3.2.28+, 3.3.18+, and 3.4.2+.
+The minimum recommended etcd versions to run in **production** are v3.4.8+ and v3.5.4+.
```

Comment:
I am thinking probably it makes more sense just to remove this in CHANGELOG-4.0, and keep the  recommendation in one place `CHANGELOG/README.md`

What do you think? @spzala

Reply:
@ahrtr yes, good point! I am not sure why didn't we remove it when modified other files but probably it was missed. Thanks!